src/compressed_tensors/quantization/quant_args.py

```diff
@@ -263,7 +309,11 @@ def round_to_quantized_type(
     """
     original_dtype = tensor.dtype
     if args.type == QuantizationType.FLOAT:
-        rounded = tensor.to(FP8_DTYPE)
+        if args.num_bits == 8:
+            rounded = tensor.to(FP8_E4M3_DATA.dtype)
+        else:
+            assert args.num_bits == 4
+            rounded = FP4_E2M1_DATA.cast_to_fp4(tensor)
```

Comment:
Same here, someone who sets num_bits==6 is going to see an assertion that it should be 4, and not either 4 or 8
```suggestion
        elif args.num_bits == 4:
            rounded = FP4_E2M1_DATA.cast_to_fp4(tensor)
        else:
            raise NotImplementedError("Only 4-bit or 8-bit allowed")
```